pandas_gbq/gbq.py

```diff
@@ -604,6 +604,7 @@ def _download_results(
     def load_data(
         self,
         dataframe,
+        project_id_table,
         dataset_id,
         table_id,
```

Comment:
I'd prefer we actually change this to pass in the table reference we create in `to_gbq`

```suggestion
        destination_table,
```